doc/telescope-file-browser.txt

```diff
@@ -78,7 +78,7 @@ fb_picker.file_browser({opts})                      *fb_picker.file_browser()*
                                         `vim.fn.expanded` automatically
                                         (default: vim.loop.cwd())
         {cwd}               (string)    dir to browse folders from,
-                                        `vim.fn.expanded` automatically 
```

Comment:
Typically, CI should trigger automatically to generate `doc/telescope-file-browser.txt`, but for some reason this only so far worked for my own PRs.

Can you please also remove the extra space in `picker.lua`?

Then it's ok to merge as lua mirror doc files perfectly.

Reply:
I removed the extra space!
70f2e11fbf70082fbb5dbcb7c140f5cf0b361af4

Reply:
Thanks :)